### DISCUSSION TITLE
New benchmark

### DISCUSSION THREAD

**Discussion by Keats** [swc-project/swc#1727] [General]
Hey

I just saw https://twitter.com/jarredsumner/status/1390084458724741121 which is a bit weird perf wise. Maybe it's worth reaching out to that person to figure out the exact benchmark? `260ms` for a single (I think?) JSX file transpilation seems odd.

**Comment by kdy1** [comment]
I'm not sure if it worths.

The author seems to not know what he is benchmarking (swc is not a command for the bundler), but I don't like the benchmark games.

**Comment by kdy1** [comment]
I made an open sourced, aligned benchmark.
Results are at

https://swc.rs/docs/benchmark-transform

**Reply by Keats** [reply]
I've seen your repo at https://github.com/kdy1/benchmark-done-right as well, I just thought it would be worth a mention as some people in the comments do believe the results from the tweet.

**Reply by kdy1** [reply]
Oh, I see.
I'll do so. Thanks!

**Reply by Keats** [reply]
Ah sorry I missed that you had already commented on it >_> It was hidden by Tweeter for some reasons.
